coreax/data.py

```diff
@@ -247,16 +78,6 @@ def normalize(self, *, preserve_zeros: bool = False) -> Data:
         return eqx.tree_at(lambda x: x.weights, self, normalized_weights)
 
 
-def as_data(x: Any) -> Data:
-    """Cast 'x' to a data instance."""
-    return x if isinstance(x, Data) else Data(x)
-
-
-def is_data(x: Any) -> bool:
-    """Return boolean indicating if 'x' is an instance of 'coreax.data.Data'."""
-    return isinstance(x, Data)
-
-
```

Comment:
missed doc issue "defining the responses of the supervised paired " -> "defining the responses of the supervised dataset paired "

Reply:
Yep happy